controllers/validator.go

```diff
@@ -54,7 +54,21 @@ func (r *DPAReconciler) ValidateDataProtectionCR(log logr.Logger) (bool, error)
 		}
 
 		// Check the Velero flags 'no-secret' or 'no-default-backup-location' are not set
```

Comment:
here the comment is wrong, right? the if checks if both (AND) are falsy values, right? (we can fix this in follow up PR, its just a comment, want change code logic)

Reply:
right. both false